client/src/components/ProfilePictureGroup/ProfilePictureGroup.css

```diff
@@ -0,0 +1,4 @@
+
+.ProfilePictureGroup {
```

Comment:
Seems like maybe we can delete this component now?

Reply:
@zakk I didn't see this component before! This seems like a better name for the InboxUserProfilesGroup, should I re-name that component?

Reply:
Sure! I agree with you